<?xml version="1.0"?>
<launch>
    <arg name="model" default="$(find s4_vive)/urdf/test.xacro"/>
    <arg name="gui" default="true" />

    <param name="robot_description" command="$(find xacro)/xacro --inorder $(arg model)" />
    <param name="use_gui" value="$(arg gui)"/>

    <node name="joint_state_publisher" pkg="joint_state_publisher" type="joint_state_publisher" />
    <node name="robot_state_publisher" pkg="robot_state_publisher" type="state_publisher" />
    <node name="rviz" pkg="rviz" type="rviz" args="-d $(find s4_vive)/config/SETTING.rviz" required="true" />

    <node name="s4_vive_tf_overlapper" pkg="s4_vive" type="s4_vive_tf_overlapper" >
        <param name="world_frame" value="link1"/>
        <param name="from_frame"  value="link2"/>
        <param name="to_frame"    value="link3"/>
    </node>    
</launch>
